internal/app/client/conformance_client.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/url"
+
+	"connectrpc.com/conformance/internal/app"
+	"connectrpc.com/conformance/internal/gen/proto/connect/connectrpc/conformance/v1alpha1/conformancev1alpha1connect"
+	v1alpha1 "connectrpc.com/conformance/internal/gen/proto/go/connectrpc/conformance/v1alpha1"
+	"connectrpc.com/connect"
+)
+
+type Invoker struct {
```

Comment:
Lint requested this to be exported since `NewInvoker` was returning an unexported type. However, this is only used within the `client` package, so we could ignore that rule for this case. Either way works for me.

Reply:
The solution is to unexport `NewInvoker`, too 😛 

In general, only export what is actually used/needed outside of the package.

Reply:
Ugh, duh.  Yeah, sorry about that. Fixed.